mmv1/third_party/terraform/services/compute/resource_compute_instance_template.go.tmpl

```diff
@@ -964,6 +964,12 @@ be from 0 to 999,999,999 inclusive.`,
 							ForceNew:    true,
 							Description: `The number of threads per physical core. To disable simultaneous multithreading (SMT) set this to 1. If unset, the maximum number of threads supported per core by the underlying processor is assumed.`,
 						},
+						"turbo_mode": {
+							Type:        schema.TypeString,
+							Optional:    true,
```

Comment:
not sure if any of these need to be computed?

Reply:
Probably not. If it did, we would expect to see some users who don't specify the value getting permadiffs. We can also verify by looking at the debug logs once the CI tests run.